meilisearch-release-changelog/v1.11.0.md

```diff
@@ -0,0 +1,208 @@
+{% if field.is_searchable and not field.value == nil %}
+{{ field.name }}: {{ field.value }}\n
+{% endif %}
+{% endfor %}
+```
+
+This last change is backward compatible. Embedders that use the v1.10 document template will keep using it. The new default document template will only be applied to newly added embedders.
+
+## Speed up indexing of your vector database with binary quantization
+
+The binary quantization converts your embeddings of floating point numbers (using 32 bits) into embeddings of boolean values (using 1 bit).
+In our benchmarks, it reduced the size of the database by a factor of 10, and divided the indexing time by a factor of 6 without much impact on the search time.
+
+> [!WARNING]
+> Enabling this feature will update all of your vectors to contain only `1`s or `-1`s, impacting the relevancy significantly.
+> Once enabled, **there is no coming back**. Before trying it out on your main index, you should try the feature on a smaller index to ensure it fits your use case.
+
+Although this feature degrades the relevancy of the semantic searches, it showed great results on collections with many dimensions (> 1300) and many points (more than 100000).
+
+Enable the binary quantization by updating the settings of your embedder with:
+```bash
+curl \
+  -X PATCH 'http://localhost:7700/indexes/movies/settings' \
+  -H 'Content-Type: application/json' \
+  --data-binary '{
+    "embedders": {
+      "image2text": {
+        "binaryQuantized": true
+      }
+    }
+  }'
+```
+
+Done by @irevoire in #4941.
+
+## Federated search improvements
+
+### Request by-index facet distribution and stats
+
+Request facet distributions and facet stats in your federated search, by using `federation.facetsByIndex` in the POST `POST /multi-search` route.
+
+By default the distribution and stats are returned for each index that participates in the federated search.
+
+```json5
+POST /multi-search
+{
+  "federation": {
+    "limit": 20,
+    "offset": 0,
+	"facetsByIndex": {
+	  "movies": ["title", "id"],
+	  "comics": ["title"],
+	}
+  },
+  "queries": [
+    {
+      "q": "Batman",
+      "indexUid": "movies"
+    },
+    {
+      "q": "Batman",
+      "indexUid": "comics"
+    }
+  ]
+}
+```
+
+### Request merged facet distribution and stats
+
+Merge the returned facets in a single facet distribution and stats that is global to the entire request, by using `federation.mergeFacets` in the POST `POST /multi-search` route.
+
+The number of returned values can be limited for all facets with `federation.mergeFacets.maxValuesPerFacet`.
+
+```json5
+POST /multi-search
+{
+
+  "federation": {
+    "limit": 20,
+    "offset": 0,
+	  "facetsByIndex": {
+	    "movies": ["title", "id"],
+	    "comics": ["title"],
+	  },
+	  "mergeFacets": {
+	    "maxValuesPerFacet": 10,
+	  }
+  }
+  "queries": [
+    {
+      "q": "Batman",
+      "indexUid": "movies"
+    },
+    {
+      "q": "Batman",
+      "indexUid": "comics"
+    }
+  ]
+}
+```
```

Comment:
````suggestion
#### Merged facet information

To obtain facet distribution and stats for all index merged into a single ,use both `facetsByIndex` and `mergeFacets` when querying the `POST` `/multi-search` endpoint:

```json5
POST /multi-search
{

  "federation": {
    "limit": 20,
    "offset": 0,
	  "facetsByIndex": {
	    "movies": ["title", "id"],
	    "comics": ["title"],
	  },
	  "mergeFacets": {
	    "maxValuesPerFacet": 10,
	  }
  }
  "queries": [
    {
      "q": "Batman",
      "indexUid": "movies"
    },
    {
      "q": "Batman",
      "indexUid": "comics"
    }
  ]
}
```

The response includes two new fields, `facetDistribution` and `facetStarts`:

```json5
{
  "hits": […],
  …
  "facetDistribution": {
    "title": {
      "Batman returns": 1
      "Batman: the killing joke": 
    },
    "id": {
      "42": 1
    }
  },
  "facetStats": {
    "id": {
      "min": 42,
      "max": 42
    }
  }
}
```
````